app/components/AddDelegateModal/index.js

```diff
@@ -333,6 +339,7 @@ class AddDelegateModal extends Component<Props> {
     } = this.getBalanceState(balance, amount, t);
     return (
       <Modal
+        onKeyDown={this.onEnterPress}
```

Comment:
{(event)=> this.onEnterPress(event.key, isDisabled)}

......

onEnterPress = (keyVal, isDisabled)=> {
if (keyVal === 'Enter' && !isDisabled) {
....
}
}